ckanext/datagovtheme/helpers.py

```diff
@@ -456,16 +456,15 @@ def convert_top_category_to_list(str_value):
 def get_bureau_info(bureau_code):
     WEB_PATH = '/fanstatic/datagovtheme/images/logos/'
     LOCAL_PATH = 'fanstatic_library/images/logos/'
+    BUREAU_CODE_URL = 'https://project-open-data.cio.gov/data/omb_bureau_codes.csv'
 
     # handle both '007:15', or ['007:15', '007:16']
     if isinstance(bureau_code, list):
       bureau_code = bureau_code[0]
 
     filepath = ckan_tmp_path + '/logos/'
-    filename = filepath + 'bureau.csv'
-    url = config.get('ckanext.geodatagov.bureau_csv.url', '')
-    if not url:
-        url = config.get('ckanext.geodatagov.bureau_csv.url_default', '')
+    filename = filepath + 'bureau-sample.csv'
+    url = BUREAU_CODE_URL
```

Comment:
```suggestion
    url = config.get('ckanext.geodatagov.bureau_csv.url', '')
    if not url:
        url = config.get('ckanext.geodatagov.bureau_csv.url_default', BUREAU_CODE_URL)
```
I think we should preserve the ability to define a custom URL for Bureau Code URL in config.
If we don't need to do that you can just avoid this suggestion